src/RadworksGovernor.sol

```diff
@@ -0,0 +1,218 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
```

Comment:
I think this can bumped to `^0.8.20` as that is what we have as the version in the `Counters.sol`

Reply:
Done: a62f2871c666020e1ef0ee90f9f0b0417c7566e2